CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.1]
+### Fixed
+- The promise returned by the `init` method now resolves only after iframe has been loaded  ([#236](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/236))
```

Comment:
```suggestion
- The promise returned by the `init` method now resolves only after the iframe has been loaded ([#236](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/236))
```

Reply:
I was about to suggest the same change, but I noticed that in the current main branch, it’s written without "the" [here](https://github.com/MetaMask/eth-ledger-bridge-keyring/blob/main/CHANGELOG.md#411).

This version looks good. Can we just ensure the other changelog is updated as well? or it's too late we already released it ?